std/complex.d

```diff
@@ -114,19 +114,20 @@ struct Complex(T)  if (isFloatingPoint!T)
     See the $(MREF std, format) and $(REF format, std,string)
     documentation for more information.
     */
-    string toString() const /* TODO: @safe pure nothrow */
+    string toString() const @safe /* TODO: pure nothrow */
     {
         import std.exception : assumeUnique;
         char[] buf;
         buf.reserve(100);
         auto fmt = FormatSpec!char("%s");
         toString((const(char)[] s) { buf ~= s; }, fmt);
-        return assumeUnique(buf);
+        auto trustedAssumeUnique(T)(T t) @trusted { return assumeUnique(t); }
```

Comment:
I know it's required for structs to not capture the stack frame, but I think for nested functions, the capture is implied. Walter?